src/adagrams.js

```diff
@@ -1,8 +1,122 @@
+// move letterPool outside of the function
+
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
+    // create a object/hash with letters and their counts
+    const letterPool = {
```

Comment:
Style note: move the object `letterPool` outside of the function but inside the module.

You can simply write 
{A: 9,
B: 2,
C: 2
...\} without quotations